File: Arrays/Python/Move_all_zeroes_to_end_of_array.py
class Solution:
	def pushZerosToEnd(self,arr, n):
    	# code here
    	count=0
        for k in range(n):
            if(arr[k]==0):
                count+=1
            else:
                arr[k-count], arr[k] = arr[k], arr[k-count]
        return arr

File: Arrays/Java/Move_all_zeroes_to_end_of_array.java
class Solution {
    void pushZerosToEnd(int[] arr, int n) {
        // code here
        int count = 0;
        for (int i=0; i<n; i++) {
            if (arr[i] == 0) {
                count += 1;
            }
            else {
                int k = arr[i-count];
                arr[i-count] = arr[i];
                arr[i] = k;
            }
        }
    }
}
